scripts/generate-changelog-entry.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Generate a changelog entry for a suggested PR by grabbing the next available
+# auto incrementing ID in GitHub. User can choose to use another PR number
+set -euo pipefail
+
+if ! command -v curl &> /dev/null
+then
+  echo "curl not be found"
+  exit 1
+fi
+
+if ! command -v jq &> /dev/null
+then
+  echo "jq not be found"
+  exit 1
+fi
+
+if ! command -v changelog-entry &> /dev/null
+then
+  echo "changelog-entry not be found"
+  exit 1
+fi
+
+current_pr=$(curl -s "https://api.github.com/repos/mongodb/terraform-provider-mongodbatlas/issues?state=all&per_page=1" | jq -r ".[].number")
+next_pr=$((current_pr + 1))
+
+echo "==> What is the new changelog entry? Suggested is $next_pr (next PR number)"
+read -r next_pr
+
+changelog-entry -pr "$next_pr" -dir ".changelog"
```

Comment:
Our guidelines [define a limited set of header types](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2123/files#diff-57e0bba85dc98868e7f9f864235d7316522b9512271bd6e4394fd3934b27bb0eR45), can we make use of the [allowed types flag](https://github.com/hashicorp/go-changelog/tree/main/cmd/changelog-entry#customizing-the-allowed-types) so we dont use any additional header types?

Reply:
Done!